<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PSSA</class>
 <widget class="QWidget" name="PSSA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patch Size Statistical Analysis (PSSA) </string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_16">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>2</number>
     </property>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_01_02">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_01_input">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLineEdit" name="lineEdit_01_input"/>
             </item>
             <item>
              <widget class="QToolButton" name="Button_01_input">
               <property name="minimumSize">
                <size>
                 <width>56</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_02_input_show">
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_02_01">
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QTableWidget" name="table_02_input_show"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_02_02">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
border-style: outset; /* 边框风格 */
border-width: 0px; /* 边框宽度 */
border-radius: 5px; /* 边框圆角 */
}
QPushButton:hover { background-color: rgb(236, 236, 236); }</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QFrame" name="frame_02_button">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::StyledPanel</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QPushButton" name="Button_02_remove">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../qrc/res.qrc">
                       <normaloff>:/svg/remove.svg</normaloff>:/svg/remove.svg</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>32</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="Button_02_up">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../qrc/res.qrc">
                       <normaloff>:/svg/up.svg</normaloff>:/svg/up.svg</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>32</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="Button_02_down">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../qrc/res.qrc">
                       <normaloff>:/svg/down.svg</normaloff>:/svg/down.svg</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>32</width>
                       <height>32</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="Button_02_check">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../qrc/res.qrc">
                       <normaloff>:/svg/check.svg</normaloff>:/svg/check.svg</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>32</width>
                       <height>32</height>
                      </size>
                     </property>
                     <property name="checkable">
                      <bool>false</bool>
                     </property>
                     <property name="autoRepeat">
                      <bool>true</bool>
                     </property>
                     <property name="autoExclusive">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_03_04">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_03_pdc_parameters">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="3,6,4">
             <item>
              <widget class="QCheckBox" name="checkBox_03_print_patch">
               <property name="text">
                <string>Print patch</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_03_calculate_gaussian_parrameters">
               <property name="text">
                <string>Calculate Gaussian parameters</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label__03_neighbor_type">
                 <property name="text">
                  <string>Neighbor type</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comBox_03_neighbor_type">
                 <property name="currentIndex">
                  <number>1</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>4</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>8</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_04_patch">
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_04_save_location">
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_04_save_location">
                  <property name="text">
                   <string>Save location</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_04_save_location"/>
                </item>
                <item>
                 <widget class="QToolButton" name="Button_04_save_location">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_04_center_data">
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_04_center_data">
                  <property name="text">
                   <string>Center data</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_04_center_data"/>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_04_center_data">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_05_run">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="Button_05_calculate">
            <property name="text">
             <string>Calculate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_06_result">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="frame_06_lognormal_distribution">
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_06_standard_deviation_2">
               <property name="text">
                <string>Lognormal distribution (cell):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_06_mean">
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="bottomMargin">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_06_mean">
                  <property name="text">
                   <string>Mean</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_06_mean">
                  <property name="minimumSize">
                   <size>
                    <width>90</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_06_standard_deviation">
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="bottomMargin">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_06_standard_deviation">
                  <property name="text">
                   <string>Standard deviation</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_06_standard_deviation">
                  <property name="minimumSize">
                   <size>
                    <width>90</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_06_power_law_distribution">
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Power-law distribution (cell):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_06_scale_constant">
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="bottomMargin">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_06_scale_constant">
                  <property name="text">
                   <string>Scale constant</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_06_scale_constant">
                  <property name="minimumSize">
                   <size>
                    <width>90</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_06_power">
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="bottomMargin">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_06_power">
                  <property name="text">
                   <string>Power</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_06_power">
                  <property name="minimumSize">
                   <size>
                    <width>90</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_07_output">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="MyTextBrowser" name="textBrowser"/>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>src.mytextbrowser</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrc/res.qrc"/>
 </resources>
 <connections/>
</ui>
